src/main/kotlin/no/nav/syfo/dialogmote/api/v2/DialogmoteApiV2.kt

```diff
@@ -43,6 +46,28 @@ fun Route.registerDialogmoteApiV2(
                 call.respond(dialogmoteDTOList)
             }
         }
+        get(dialogmoteApiVeilederIdentUrlPath) {
+            val callId = getCallId()
+            val token = getBearerHeader()
+                ?: throw IllegalArgumentException("No Authorization header supplied")
+
+            val dialogmoteDTOList =
+                dialogmoteService.getDialogmoteList(veilederIdent = getNAVIdentFromToken(token))
+                    .map { dialogmote -> dialogmote.toDialogmoteDTO() }
+
+            val identList =
+                dialogmoteDTOList.map { dialogmoteDTO -> PersonIdentNumber(dialogmoteDTO.arbeidstaker.personIdent) }
+
+            if (dialogmoteTilgangService.hasAccessToAllDialogmotePersons(
+                    personIdentNumberList = identList,
+                    token = token,
+                    callId = callId
+                )
+            ) {
+                call.respond(dialogmoteDTOList)
+            }
+        }
```

Comment:
`hasAccessToAllDialogmotePersons` skal ikke brukes her. 
som funksjonen heter så krever den at man har tilgang til alle møtene i listen for å få noe ut.
Dvs at dersom man har 100 møter og kun 1 av de har man mistet tilgang til, så får man ingen av de 99 andre ut.
I dette tilfellet så skal man altså få ut de 99 andre, aka alle møtene man har tilgang til og hører til seg.